src/styles/variables.scss

```diff
@@ -0,0 +1,5 @@
+:root {
+  --main-accent: #060b35;
+  --secondary-accent: #616070;
+  --blue-accent: #00acdc;
```

Comment:
You're using CSS variables instead of SCSS variables. In SCSS variables should be defined at the top of the SCSS file using $ sign.